pkg/etcdenvvar/etcd_env_test.go

```diff
@@ -0,0 +1,82 @@
+package etcdenvvar
+
+import (
+	"reflect"
+	"testing"
+
+	configv1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/library-go/pkg/crypto"
+)
+
+func TestWhitelistEtcdCipherSuites(t *testing.T) {
```

Comment:
I'm a bit confused by the intent of this test. What future change would cause it to fail, and what would the consequence of that failure be if uncaught?

Reply:
yeah, I think it's just exercising the code I added to ensure the logical path was valid with ommissions. Understanding the net ciphers that we honor is the intent. For example, I would expect the results to change if we bumped etcd to 3.5 in which case  tls1.3 test would fail as an example because we would gain that support.

If the supported ciphers change we would want to know is the short answer. I can reduce the test coverage to only test 1.2 and v1.3 `TLSProfileModernType` vs `TLSProfileModernType` v1.2.

Reply:
Maybe document that intent? Otherwise the test as written might be perceived by the uninitiated to be testing code we don't own.

Reply:
makes sense

Reply:
I adjusted the test to only test the actual profile we use as default and documented the test's intent.